ui/src/components/AutoCompleteText/index.jsx

```diff
@@ -184,7 +187,7 @@ function AutoCompleteText({
                 undefined
               ),
               endAdornment: (
-                <div>
+                <div className={classes.endAdornment}>
```

Comment:
What is this accomplishing? I don't see any visual differences with or without it.

Reply:
I added this class to change the cursor behavior (pointer) when hovering the endAdornment for `TextField` (the dropDown Icon) to indicate that it is a clickable element.
```javascript
    endAdornment: {
    cursor: 'pointer',
    },
```
A visual difference can be seen when hovering the dropDown icon

@bhearsum

Reply:
Ah, thank you! I overlooked that.